pkg/sql/parser/builtins.go

```diff
@@ -366,6 +366,20 @@ var Builtins = map[string][]Builtin{
 		},
 	},
 
+	"gen_random_uuid": {
+		Builtin{
+			Types:      ArgTypes{},
+			ReturnType: fixedReturnType(TypeUUID),
+			category:   categoryIDGeneration,
+			impure:     true,
+			fn: func(_ *EvalContext, _ Datums) (Datum, error) {
+				uv := uuid.MakeV4()
+				return NewDUuid(DUuid{uv}), nil
+			},
+			Info: "Generates a random UUID and returns its byte string representation.",
```

Comment:
Sorry to ask for another change, but could we also update this description, since it doesn't return the byte string any more?

Reply:
Oh, shame on me, just a minute.

Reply:
I explicitly specified the return type to avoid confusion with BYTES type.